Gems/Atom/Feature/Common/Assets/Shaders/PostProcessing/Taa.azsl

```diff
@@ -247,7 +247,7 @@ void MainCS(
     float3 sdFromLastFrame = standardDeviationWeight / abs(lastFrameClampedColor.r - thisFrameColor.r);
 
     float currentFrameWeight = PassSrg::m_currentFrameContribution;
-    currentFrameWeight *= saturate(sdFromLastFrame * sdFromLastFrame);
+    currentFrameWeight *= saturate(sdFromLastFrame * sdFromLastFrame).x;
```

Comment:
The real issue here is `sdFromLastFrame` should be a `float`

Reply:
Changed `sdFromLastFrame` to `float`